### ISSUE TITLE
Linter idea: SpaceAfterVariableColon

### ISSUE THREAD

**Issue by srawlins** [brigade/scss-lint#445]
When I lint a.scss:

```scss
$my-color:     #fff;
```

I'd like a linter to print a warning like: "a.scss:1 [W] SpaceAfterVariableColon: Colon after a variable should be followed by one space." (Right now, scss-lint is quiet.)

This would support the same styles as [SpaceAfterPropertyColon](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceafterpropertycolon), except the `aligned` one, I think...

I'd be happy to write this.

**Comment by lencioni** [comment]
:+1: This makes sense to me! You should probably implement SpaceAfterVariableName a la [SpaceAfterPropertyName](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceafterpropertyname) at the same time.

**Comment by srawlins** [comment]
Just so I don't mislead anyone: I would still be happy to write this, but it's not going to happen soon. I'm on vacation through the end of June.

**Comment by sds** [comment]
Thanks for offering to implement this, @srawlins. Happy to merge a pull request when you get around to it.

**Comment by sds** [comment]
Implemented by #498. Thanks!

**Comment by kaelig** [comment]
@sds looks like this was actually never implemented, unless it was removed at some point?

**Comment by sds** [comment]
Hey @kaelig, this linter definitely exists: https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#spaceaftervariablename

Which version of SCSS-Lint are you running?

**Comment by kaelig** [comment]
@sds yes, but SpaceAfterVariableColon doesn't exist.

**Comment by sds** [comment]
Whoops, nevermind. Different location that linter is checking. :)

Reopening.

**Comment by kaelig** [comment]
Cheers!

**Comment by sds** [comment]
Implemented in #711
